apps/web/components/run-response-metadata.tsx

```diff
@@ -0,0 +1,23 @@
+import { Separator } from "./ui/separator";
+
+export const RunResponseMetadata = ({
+  title,
+  value,
+  hideSeparator = false,
+}: {
+  title: string;
+  value?: number | string;
+  hideSeparator?: boolean;
+}) => {
+  if (!value) {
+    return;
+  }
+  return (
+    <>
+      {!hideSeparator && <Separator orientation="vertical" className="h-6" />}
+      <p className="text-sm text-muted-foreground">
```

Comment:
is this the same font size as the output text? if so, it should ideally be a bit smaller.

Reply:
Before: 

<img width="896" alt="image" src="https://github.com/empirical-run/empirical/assets/57426646/3e38ff95-a0d2-4bf8-835a-938011a6e195">

After:

<img width="897" alt="image" src="https://github.com/empirical-run/empirical/assets/57426646/76a9fc51-d114-4804-9813-fa9b783b3ba5">

Reduced it